controllers/depresolver/depresolver_test.go

```diff
@@ -459,53 +533,98 @@ func TestResolveConfigWithInvalidIpAddressEdgeDnsServer(t *testing.T) {
 	// arrange
 	defer cleanup()
 	expected := predefinedConfig
-	expected.EdgeDNSServer = "22.147.90.2."
+	expected.EdgeDNSServers = []utils.DNSServer{
+		{
+			Host: "22.147.90.2.",
```

Comment:
Let's use more dummy ip here please

Reply:
`1.2.3.4`? or CSI-Miami-lvl dummy :D

Reply:
I will wait for https://github.com/k8gb-io/k8gb/pull/616 to go in and then resolve the conflicts

Reply:
done